src/js/util/ame.js

```diff
@@ -95,6 +95,23 @@ export default function() {
         return !e.getFilterableText().match(regex);
       },
     },
+    BTD_user_regex: {
+      display: {
+        global: true,
+      },
+      name: 'Username (Regular Expression)',
+      descriptor: 'usernames matching',
+      placeholder: 'Enter a regular expression',
+      function(t, e) {
+        const regex = new RegExp(t.value, 'g');
+
+        if (e.user === undefined) {
+          return true;
+        }
```

Comment:
Do we know if `e.user` can be, like, `null` or something that's not `undefined` but still falsy?

Reply:
nope, it is always null, because the whole structure of `e` is different depending on what's filtered.

any call on `e.user` busts timelines that are notifications/messages, pretty much.

I could adjust the check to be `if (!e.user) return true;` at the very top, like we do it with all the other filters though.